libs/core/src/lib/utils/services/keyboard-support/keyboard-support.service.ts

```diff
@@ -38,13 +52,19 @@ export class KeyboardSupportService<T> {
     onKeyDown(event: KeyboardEvent): void {
```

Comment:
I think try refactoring this function to 

```suggestion
    /** @hidden */
    onKeyDown(event: KeyboardEvent): void {
        this._keyManager.onKeydown(event);
        if (KeyUtil.isKeyCode(event, TAB) && this._focusTrapOnTab) {
            event.preventDefault();
            hasModifierKey(event, 'shiftKey') ? this.keyManager.setPreviousItemActive() : this.keyManager.setNextItemActive();
        }
    }
```